expression-language/src/main/scala/io/zeebe/el/impl/feel/FeelEvaluationResult.scala

```diff
@@ -7,12 +7,17 @@
  */
 package io.zeebe.el.impl.feel
 
-import java.lang
+import java.{lang, util}
 
 import io.zeebe.el.{EvaluationResult, Expression, ResultType}
+import io.zeebe.util.buffer.BufferUtil
+import io.zeebe.util.buffer.BufferUtil.cloneBuffer
 import org.agrona.DirectBuffer
 import org.camunda.feel.syntaxtree.{Val, ValBoolean, ValContext, ValList, ValNull, ValNumber, ValString}
 
+import scala.collection.JavaConverters._
+import scala.collection.mutable.ListBuffer
```

Comment:
Optimized imports by Idea:

```suggestion
import java.{lang, util}

import io.zeebe.el.{EvaluationResult, Expression, ResultType}
import io.zeebe.util.buffer.BufferUtil.cloneBuffer
import org.agrona.DirectBuffer
import org.camunda.feel.syntaxtree.{Val, ValBoolean, ValContext, ValList, ValNull, ValNumber, ValString}

import scala.jdk.CollectionConverters._
```